gen3-client/g3cmd/upload-multiple.go

```diff
@@ -77,22 +79,45 @@ func init() {
 			}
 
 			furObjects := validateObject(objects, uploadPath)
-
 			if batch {
 				workers, respCh, errCh, batchFURObjects = initBatchUploadChannels(numParallel, len(objects))
 			}
 
 			for i, furObject := range furObjects {
-				if batch {
+				if forceMultipart {
+					uploadPath := []string{furObjects[i].FilePath}
+					_, multipartFilePaths := validateFilePath(uploadPath, forceMultipart)
```

Comment:
validateFilePath returns two arrays (singlepartFilePaths, multipartFilePaths). It looks as if the `singlepartFilePaths` array is ignored. @BinamB